server/src/main/java/io/crate/analyze/AnalyzedAlterTableDropColumn.java

```diff
@@ -38,4 +43,16 @@ public <C, R> R accept(AnalyzedStatementVisitor<C, R> visitor, C context) {
     @Override
     public void visitSymbols(Consumer<? super Symbol> consumer) {
     }
+
+    public record DropColumn(Reference ref, boolean ifExists) {
```

Comment:
maybe extract it to it's own file?

Reply:
Either way is fine to me. 

Does PG support dropping multiple columns with mixed exists flag/no flag?
Is smth like `DROP COLUMN a IF EXISTS, DROP COLUMN b `valid?

Otherwise I thought we could keep using `List<Reference>` and have a single flag.

Reply:
yep it supports it:
```
matriv=> create table t(a int, b int, c int, d int);
CREATE TABLE
matriv=> alter table t drop if exists b, drop c, drop if exists e;
NOTICE:  column "e" of relation "t" does not exist, skipping
ALTER TABLE
matriv=> select * from t;
 a | d 
---+---
(0 rows)
```